StatusEffect/InHeat.gd

```diff
@@ -5,7 +5,18 @@ func _init():
 	isBattleOnly = false
 
 func getEffectName():
-	return "In Heat"
+	var toCheck = [
+		character.hasBodypart(BodypartSlot.Penis),
+		character.hasBodypart(BodypartSlot.Vagina)
+	]
+	
+	match toCheck:
+		[false, true]:
+			return "In Heat"
+		[true, false]:
+			return "In Rut"
+		[true, true]:
+			return "In Heat and In Rut"
```

Comment:
This is a strange way to check that, I would prefer just lots of if-elif checks.

But if you want rut its better to make it its own status effect. Right now its just renamed heat and works exactly like it (that means only girls have it)